CIP-????/README.md

```diff
@@ -0,0 +1,156 @@
+---
+CIP: ?
```

Comment:
```suggestion
CIP: "?"
```
We've found out by trial & error that quoting this character prevents the YAML error on GitHub preview `mapping keys are not allowed in this context at line 1 column 6` but haven't gotten this into the documentation & templates yet.